desktop/conf/pseudo-distributed.ini.tmpl

```diff
@@ -341,6 +341,11 @@
     # e.g. <image xlink:href="/static/desktop/art/hue-logo-mini-white.png" x="0" y="0" height="40" width="160" />
     ## logo_svg=
 
+    # Hue uses Localstorage to keep the users settings and database preferences.
```

Comment:
Move out of `[[custom]]` to `[desktop]`? (as not a custom visual change. not sure why we put `cacheable_ttl` there too)

e.g. below `enable_link_sharing=true`?